docs/docs/migration-guide.mdx

```diff
@@ -437,6 +437,40 @@ response_selector_output.get("default")
                         .get("intent_response_key")
 ```
 
+### Core Featurization
+
+* `LabelTokenizerSingleStateFeaturizer` is deprecated.
+  Please use `SingleStateFeaturizer` with appropriate nlu pipeline.
+  To replicate `LabelTokenizerSingleStateFeaturizer` functionality,
+  add a `Tokenizer`  with `intent_tokenization_flag: True` and `CountVectorsFeaturizer`
+  to the NLU pipeline. Please update your configuration file.
+
+  For example:
+  ```yaml
+  language: en
+  pipeline:
+    - name: WhitespaceTokenizer
+      intent_tokenization_flag: True
+    - name: CountVectorsFeaturizer
+  ```
+
+  Please train both NLU and Core (using `rasa train`) to use a trained tokenizer and
+  featurizer for core featurization.
+
+* `BinarySingleStateFeaturizer` is deprecated and will be removed in the future.
+  Please use `SingleStateFeaturizer` with nlu pipeline where `intent_tokenization_flag`
```

Comment:
`nlu` -> `NLU`

Reply:
"with nlu pipeline" might not be very clear. Would mention as above that you need to train your bot with `rasa train`.

Reply:
rephrased